### ISSUE TITLE
[no-duplicates] invalid fix when there are 3 imports with named imports and dangling commas

### ISSUE THREAD

**Issue by bradzacher** [import-js/eslint-plugin-import#2840]
**Input:**

```js
import {A1,} from 'foo';
import {B1,} from 'foo';
import {C1,} from 'foo';

import {
  A2,
} from 'bar';
import {
  B2,
} from 'bar';
import {
  C2,
} from 'bar';
```

**Expected:**

Error and fixed to
```js
import {A1,B1,C1,} from 'foo';
import {
  A2,
  B2,
  C2,
} from 'bar';
```
(or something similar)

**Actual:**

Error and fixed to
```js
import {A1,B1,,C1,} from 'foo';
              ^ SYNTAX ERROR
import {
  A2,

  B2,
,
^ SYNTAX ERROR
  C2,
} from 'bar';
```

---

Interestingly this bug requires 3 (or more) duplicate declarations to trigger.

I haven't looked yet - but given it happens for 3 and not 2 - I'd assume that ESLint is merging the distinct fixes poorly - leading to the syntax error.

**Comment by AngusMacrae** [comment]
facing this bug too, can confirm it only occurs with 3 or more duplicate import paths

**Comment by ljharb** [comment]
Tests pass on latest master with this snippet, so I'll close this shortly when I push those up.

**Comment by bradzacher** [comment]
@ljharb the ESLint rule tester might not pick this up? Depends how it composites fixes together I guess. Might be worth a quick whirl as a manual test against a file with the CLI to triple check.

**Comment by ljharb** [comment]
I’ve not run into any cases like that with the rule tester; could you make a small repro repo?

**Comment by bradzacher** [comment]
Hmm - investigating more this appears to be working fine now yeah.
Some combination of the latest ESLint version and the latest plugin version appears to have fixed it.
